src/pages/Order/Order.js

```diff
@@ -0,0 +1,286 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const Order = () => {
+  const [orderList, setOrderList] = useState([]);
+  const [openList, setOpenList] = useState(false);
+  const [orderInfo, setOrderInfo] = useState([]);
+  const tokenOrder = localStorage.getItem('token', orderList.token);
```

Comment:
```suggestion
  const tokenOrder = localStorage.getItem('token');
```
로컬 스토리지에 있는 토큰을 가져오는 거면 이렇게 토큰에 대한 키만 입력해서 가져오면 됩니다!